plugins/redis/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+
+# Component: redis
+
+#####################################################
+# Description:
+# ------------
+#    Cmake Input File for redis
+#####################################################
+
+project( redis )
+
+if (USE_REDIS)
+  ADD_PLUGIN(redis PACKAGES REDIS OPTION MAKE_REDIS)
+  if ( MAKE_REDIS )
+    set (    SRCS
+             redisplugin.hpp
+             redissync.hpp
+
+             redisplugin.cpp
+             redissync.cpp
+        )
+
+    include_directories (
+             ./../../system/include
+             ./../../rtl/eclrtl
+             ./../../rtl/include
+             ./../../common/deftype
+             ./../../system/jlib
+             ${LIBHIREDIS_INCLUDE_DIRS}
+        )
+
+    ADD_DEFINITIONS( -D_USRDLL -DECL_REDIS_EXPORTS)
+
+    HPCC_ADD_LIBRARY( redis SHARED ${SRCS} )
+    if (${CMAKE_VERSION} VERSION_LESS "2.8.9")
+      message("WARNING: Cannot set NO_SONAME. shlibdeps will give warnings when package is installed")
+    else()
```

Comment:
instead. It's a new change I made to all the plugins that set NO_SONAME a couple of days ago (as it does not work on apple)